.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```

Comment:
I think we should merge these two, because they will probably never both apply. For example, in the ISSUE_TEMPLATE in the root we have:

```
    Environment: (Node.js/browser/hybrid app/etc.)
    Platform: (Chrome/FF/Safari/Edge/iOS/Android/etc.)
    Adapter: (idb/indexeddb/memory/leveldb/etc.)
    Server: (CouchDB/Cloudant/PouchDB Server/etc.)
```

Adapter is particularly important.